packages/components/src/internal/components/domainproperties/validation/RegexValidationOptions.tsx

```diff
@@ -216,12 +136,64 @@ export class RegexValidationOptions extends React.PureComponent<RegexValidationO
                             'Error Message',
                             'The message that will be displayed to the user in the event that validation fails for this field.'
                         )}
-                        {this.renderFailValidationCheckbox(validator.properties.failOnMatch)}
-                        {this.renderName(validator.name)}
-                        {this.renderRemoveValidator()}
+                        <Row>
+                            <Col xs={this.labelWidth} />
+                            <Col xs={this.fieldWidth} className="domain-validation-failOnMatch-row">
+                                <Checkbox
+                                    id={createFormInputId(DOMAIN_VALIDATOR_FAILONMATCH, domainIndex, validatorIndex)}
+                                    name={createFormInputName(DOMAIN_VALIDATOR_FAILONMATCH)}
+                                    checked={validator.properties.failOnMatch}
+                                    onChange={this.onChange}
+                                >
+                                    Fail validation when pattern matches field value
+                                    <LabelHelpTip title="Fail when pattern matches?">
+                                        By default, validation will fail if the field value does not match the specified
+                                        regular expression. Check this box if you want validation to fail when the
+                                        pattern matches the field value.
+                                    </LabelHelpTip>
+                                </Checkbox>
```

Comment:
Are we offering this option for the TextChoice fields as well?  Seems odd.

Reply:
Nope, we aren't offering this in the UI in any way. If you view the JSON for a text choice validator it will have this property since it is included in the default validator definition, but it has no impact on the text choice case.